saplings/profile/src/Profile.js

```diff
@@ -58,8 +58,8 @@ export function Profile() {
           );
           setKeys(JSON.parse(userKeys).data);
         } catch (err) {
-          switch (err.code) {
```

Comment:
Typo in commit message: s/componenet/component

Reply:
fixed